index.js

```diff
@@ -0,0 +1,16 @@
+
+/**
+ * Dependencies
+ */
+
+var config = require('./config');
+var app = require('./app');
+
+// start listening
+const port = process.env.PORT || config.port;
+const address = process.env.ADDRESS || config.address || '';
+var server = app.listen(port, address, function () {
+  var host = server.address().address;
+  var port = server.address().port;
+  console.log(`magnet service listening at http://${host}:${port}`);
```

Comment:
Moved the starting of the server out of `app.js` so that is didn't run during test.